docs/playbooks/frameworks/wagmi.md

```diff
@@ -33,14 +46,55 @@ npm create vite@latest starter-app -- --template react
 
 ### Wagmi
 
-We'll use a few different files to properly set everything up. Let's start with the first—create a file called `wagmi.js` to handle chain configurations. You should first create a `.env` that handles things like API keys. Our example with use Alchemy for the provider URL and also use WalletConnect's API for the connector. The file should contain:
+We'll use a few different files to properly set everything up. Let's start with the first—create a file called `wagmi.js` to handle chain configurations. You should first create a `.env` that handles things like API keys. Our example uses a public provider URL and will also use WalletConnect's API for the connector. The file should contain:
 
 ```txt title=".env"
-VITE_ALCHEMY_API_KEY=your_key
 VITE_WALLET_CONNECT_PROJECT_ID=your_key
+ENABLE_TESTNETS=true
+```
+
+Note that we'll use `import.meta.env` to access these variables (below) in the `wagmi.js` file. Depending on your frontend setup, it may differ. With Vite, you use `import.meta.env` and prefix the variable with `VITE_`. But, if you're using Next.js, you'd use `process.env` and prefix with `NEXT_PUBLIC_`.
+
+<Tabs groupId="sdk">
+<TabItem value="jsv2" label="wagmi v2">
+
+```jsx title="src/wagmi.js"
+import "@rainbow-me/rainbowkit/styles.css";
```

Comment:
idk if helpful, but if the Studio ever needs to adopt wagmi v2, this should explain the bulk of it. i just didnt' pass custom RPC providers and opted for wallet-only RPCs.